src/stylesheets/elements/_tags.scss

```diff
@@ -89,29 +89,46 @@
   border: 1px solid #981b1e;
   background-color: #e59393;
 }
+//Border colors
+.sds-tag--border-blue {
+  border-color:  #00a6d2!important;
+  border-radius: 3px;
+}
+.sds-tag--border-red {
+  border-color:  #981b1e!important;
+  border-radius: 3px;
+}
+.sds-tag--border-yellow {
+  border-color:  #fdb81e!important;
+  border-radius: 3px;
+}
 // Outline tag
-
-.sds-tag[class*='sds-tag--outline'] {
-  white-space: nowrap;
-  text-align: center;
-  @include u-font('sans', '3xs');
-  @include u-text('base-dark');
+.sds-tag[class*="sds-tag--outline"] {
+  @include u-font("sans", "3xs");
+  @include u-text("base-dark");
   line-height: 20px;
   font-weight: 550;
   padding: 5px;
   border: 1px solid;
   margin-right: 5px;
-  display: inline-block;
+  display: flex;
+  justify-content: center;
+  align-items: center;
   width: 115px;
   background-color: #f5f5f0;
+  height: calc(100% - 1.3rem);
```

Comment:
Be careful when using calc there are known issues in IE11. This implementation seems to work fine though.

https://caniuse.com/#feat=calc

Reply:
No longer using calc